docs/content/docs/updates/pre-v1.0.0.md

```diff
@@ -239,6 +239,6 @@ better reflect their experimental state. If you use these classes, you can add a
 ### Other notable changes
 
 - The published {{< apiref "fluid-framework" >}} package now publishes ESNext modules ({{< issue 7474 >}}).
-- Various APIs in the {{< apiref "azure-client" >}} and {{< apiref "tinylicious-client" >}} now return {{< apiref
-  "IFluidContainer" >}}s instead of {{< apiref "FluidContainer" >}}s. This change should have no effect to developers
-  since the runtime object is the same ({{< issue 7457 >}}).
+- Various APIs in the {{< apiref "azure-client" >}} and {{< apiref "tinylicious-client" >}} now return {{< relref
+  "/docs/apis/fluid-framework/ifluidcontainer.md" >}}s instead of {{< relref "/docs/apis/fluid-framework/fluidcontainer.md" >}}s.
+  This change should have no effect to developers since the runtime object is the same ({{< issue 7457 >}}).
```

Comment:
Ideally the api-ref shortcode would continue to work.

Reply:
Actually I think this will just output a bare URL which probably isn't what you want.

Reply:
Where `apiRef` defined? A brief search didn't reveal anything. Maybe its definition just needs to be updated?

Reply:
It's here: https://github.com/microsoft/FluidFramework/blob/main/docs/layouts/shortcodes/apiref.html

Shortcode docs here: https://gohugo.io/content-management/shortcodes/#readout

Reply:
Awesome! Thank you!